src/R/Editor/Impl/Completions/RCompletionContextExtensions.cs

```diff
@@ -30,12 +38,20 @@ public static bool IsCaretInNamespace(this IEditorView view, out bool tripleColo
         }
 
         public static bool IsCaretInLibraryStatement(this IRIntellisenseContext context)
-            => context.Session.View.IsCaretInLibraryStatement();
+            => context.Session.View.IsCaretInLibraryStatement(context.EditorBuffer);
```

Comment:
Is it possible that `EditorBuffer` is null?

Reply:
Nope.